src/test/ui/moves/moves-based-on-type-match-bindings.stderr

```diff
@@ -8,6 +8,10 @@ LL |     touch(&x);
    |           ^^ value borrowed here after partial move
    |
    = note: partial move occurs because `x.f` has type `String`, which does not implement the `Copy` trait
+help: consider cloning `x.f`
+   |
+LL |         Foo {f.clone()} => {}
+   |               ++++++++
```

Comment:
This seems to be an incorrect suggestion. We'll need to check that we're not in a pattern, we can only call function in expressions.

Reply:
Yes, but I couldn't figure a way to check :')